static/js/components/PersonalTab.js

```diff
@@ -69,6 +78,9 @@ class PersonalTab extends React.Component {
         placeholder="Preferred language"
       /><br />
 
+      <Button raised onClick={this.saveAndContinue.bind(this)}>
+        Save and continue
+      </Button>
```

Comment:
could we maybe gray this out until the user has made any edits to their profile?

maybe diff `profile` and `profileCopy`? not sure if there's an easy way to do that.

alternatively, there could just be a `profileHasBeenSaved` boolean that we set to `false` anytime we dispatch `updateProfile`, and we set to `true` anytime `saveProfile` comes back from the server successfully.

that might be too much extra work, so feel free to ignore that suggestion :P